examples/formatjson/formatjson.d

```diff
@@ -7,7 +9,7 @@ import iopipe.buffer;
 
 void main(string[] args)
 {
-    auto parser = openDev(args[1]).bufd.assumeText.jsonTokenizer!false;
+    auto parser = File(args[1]).refCounted.bufd.assumeText.jsonTokenizer;
```

Comment:
I was going to say this needs the `false` equivalent for replacing escapes, but I'm realizing I changed the default!

Ah well, I think it's better this way anyway.